src/main/java/core/basesyntax/figure/Figure.java

```diff
@@ -0,0 +1,19 @@
+package core.basesyntax.figure;
+
+import core.basesyntax.interfaces.AreaCalculator;
+import core.basesyntax.interfaces.Drawable;
+import core.basesyntax.suppliers.Color;
+
+public abstract class Figure implements AreaCalculator, Drawable {
+    private Color color;
+
+    public Figure(Color color) {
+        this.color = color;
+    }
+
+    public Color getColor() {
+        return color;
+    }
+
+    public abstract void draw();
```

Comment:
Read the comments more carefully, this method should also be removed, it is available from the interface